### PR TITLE
changing the background color

### PR DESCRIPTION
**Pull request by throne1986**
changed the background color

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@ body{
     margin: 0%;
     padding: 0px;
     // height: 100vh;
-    // background-color: lightgray;
+    background-color: aliceblue;
```
Comment (LodiLofa) [inline src/App.scss:5]: looks good

**Review by LodiLofa — COMMENTED** [review]
I think it will be better if you remove those commented styles

**Review by LodiLofa — APPROVED** [review]
well done

**Review by throne1986 — COMMENTED** [review]
looks so good

File: src/App.scss (hunk 1 above)
Comment (LodiLofa) [reply src/App.scss:5]: you can merge now

**Review by LodiLofa — APPROVED** [review]
all good

**Review by throne1986 — COMMENTED** [review]
good

**Pull request merged**